src/Apps/Order/Routes/Status/index.tsx

```diff
@@ -17,42 +17,123 @@ export interface StatusProps {
 }
 
 export class StatusRoute extends Component<StatusProps> {
+  stateCopy = () => {
+    const { state, requestedFulfillment } = this.props.order
+    switch (state) {
+      case "SUBMITTED":
+        return "Your order has been submitted."
+      case "APPROVED":
+        return "Your order is confirmed."
+      case "FULFILLED":
+        return requestedFulfillment.__typename === "Ship"
+          ? "Your order has shipped."
+          : "Your order has been picked up."
+      case "CANCELED":
+        return "Your order was canceled and refunded."
+    }
+  }
+
+  messageCopy = () => {
+    const { order } = this.props
+    const { state, requestedFulfillment } = order
+    switch (state) {
+      case "SUBMITTED":
+        return (
+          <>
+            Thank you for your purchase. You will receive a confirmation email
+            within 2 days.
+          </>
+        )
+      case "APPROVED":
+        return requestedFulfillment.__typename === "Ship" ? (
+          <>
+            Thank you for your purchase. You will be notified when the work has
+            shipped, typically within 5–7 business days.
+          </>
+        ) : (
+          <>
+            Thank you for your purchase. A specialist will contact you within 2
+            business days to coordinate pickup.
+          </>
+        )
+      case "FULFILLED":
+        const fulfillment =
+          order.lineItems.edges[0].node.fulfillments.edges[0].node
+        if (!fulfillment) {
+          return false
+        }
+        return requestedFulfillment.__typename === "Ship" ? (
+          <>
+            <>Your work is on its way.</>
+            <br />
+            <br />
+            {fulfillment.courier && (
+              <>
+                Shipper: {fulfillment.courier} <br />
+              </>
+            )}
+            {fulfillment.trackingId && (
+              <>
+                <>Tracking Info: {fulfillment.trackingId}</>
+                <br />
+              </>
+            )}
+            {fulfillment.estimatedDelivery && (
+              <>Estimated delivery: {fulfillment.estimatedDelivery}</>
+            )}
+          </>
+        ) : (
+          false
+        )
+      case "CANCELED":
+        return (
+          <>
+            The work is no longer available.
```

Comment:
I need to grab this reason from somewhere, but I'm not exactly sure where the data should come from.

Reply:
Is the intent there to show the specific reason? Or always this message (this seems generic enough)? 

If the former, I think we can get it from exchange but I'm not sure how human-readable it is.

Reply:
Yeah, that's what notion says about the copy. 

```
<Reason>

Please allow 5–7 business days for the refund to appear on your bank statement.
```

I haven't seen a way to get it currently... unless it returns as an error or something?

Reply:
This isn't actually implemented yet. I'm going to note it in the ticket and we'll just need to create a follow to wire it up after the implementation is complete.